hypershift-operator/controllers/hostedcluster/metrics/metrics_test.go

```diff
@@ -765,6 +765,102 @@ func TestReportDeletingDuration(t *testing.T) {
 	}
 }
 
+func TestReportEtcdManualInterventionRequired(t *testing.T) {
+	wrapExpectedValueAsMetric := func(expectedValue float64) *dto.MetricFamily {
+		return &dto.MetricFamily{
+			Name: pointer.String(EtcdManualInterventionRequiredMetricName),
```

Comment:
pointer.String is deprecated already, I recommend you to move to `k8s.io/utils/ptr` lib with ptr.To()